File: 2C++/Hardware.cpp
class Hardware : public Product{

    //atribut
    private:
        string brand;
        string model;
    
    public:
        //konstruktor 
        Hardware(){
            this->brand = "";
            this->model = "";
        }

    
    //setter & getter brand
    void setbrand(string brand){
        this->brand = brand;
    }
    string getbrand(){
        return this->brand;
    }

    //setter & getter model
    void setmodel(string model){
        this->model = model;
    }
    string getmodel(){
        return this->model;
    }
   
     ~Hardware(){
    }
};

File: 1Java/Hardware.java
public class Hardware extends Product{
    //atribut
    private String brand;
    private String model;
     
    //konstruktor 
    public Hardware(){
    }

    //setter & getter brand
    public void setbrand(String brand){
        this.brand = brand;
    }
    public String getbrand(){
        return this.brand;
    }

    //setter & getter model
    public void setmodel(String model){
        this.model = model;
    }
    public String getmodel(){
        return this.model;
    }
}
